test/engine/SpatialJoinAlgorithmsTest.cpp

```diff
@@ -246,7 +252,7 @@ class SpatialJoinParamTest
   void testDiffSizeIdTables(SJ task, bool addLeftChildFirst,
                             Rows expectedOutput, Row columnNames,
                             bool bigChildLeft) {
-    auto qec = buildTestQEC(std::get<4>(GetParam()));
+    auto qec = buildTestQEC(std::get<4>(GetParam()), std::get<5>(GetParam()));
```

Comment:
can you make a member function with a useful name that does `get<5>(GetParam())` same for 1 - 4, because nobody knows at this point , what the many parameters mean.